gulp/tasks/styles.js

```diff
@@ -99,6 +99,9 @@ function stylesIE8() {
  * @returns {PassThrough} A source stream.
  */
 function stylesOnDemand() {
+  const browserList = environment.getSupportedBrowserList( 'stylesModern' );
+  browserList.push( 'Explorer >= 8' );
```

Comment:
I would not modify what's returned from `getSupportedBrowserList`, what's the point of centralizing it then if it can be modified elsewhere?

Reply:
Because these should be going away as we move everything to Wagtail. Rather than dirty the source, I'd rather limit the modifications to where they belong.

Reply:
There are pages that use an on-demand header that will never be in Wagtail, like what's in Salesforce.

Reply:
Does that mean the Salesforce pages are loading all of Capital Framework twice?